src/styles/header.css

```diff
@@ -0,0 +1,40 @@
+#Header {
```

Comment:
Nice if the ids are consistent, either camelCase, PascalCase or kebab-case. I don't remember how the names are in the other files, but maybe look them up and follow the same convention? In general, we should decide a convention to follow.

Reply:
I looks like everyone except for this one is camelCase, so i think thats what we're going for 😅